docs/customization.md

```diff
@@ -95,7 +112,6 @@ The ignore file allows for better control over overwriting existing files than t
 Examples:
 
 ```sh
-# OpenAPI Generator Ignore
```

Comment:
This shouldn't be removed, either. It's common for files to include a header explaining what the file is.